src/platform/forms-system/src/js/components/FieldTemplate.jsx

```diff
@@ -24,9 +31,6 @@ export default function FieldTemplate(props) {
     Object.keys(formContext.touched).some(touched => id.startsWith(touched));
   const hasErrors =
     (formContext.submitted || isTouched) && rawErrors && rawErrors.length;
-  const requiredSpan = required ? (
-    <span className="schemaform-required-span">(*Required)</span>
-  ) : null;
```

Comment:
I replaced this const with a functional component in an effort to reduce clutter within the `<FieldTemplate>` component and make the markup more declarative by using components.